### PR TITLE
Add logging configuration module, set maven plugin versions, add new OscMain class

### PR DESCRIPTION
**Pull request by swaroopar**
1. Added new logging module
2. Added versions to maven plugins
3. added new Main class

### CODE REVIEW

File: services/api/src/main/java/org/eclipse/osc/services/api/OrchestratorApi.java [hunk 1]
```diff
@@ -33,21 +36,16 @@ public Response fetch(@HeaderParam("ocl") String oclLocation) throws Exception {
     @Path("/health")
     @GET
     @Produces(MediaType.TEXT_PLAIN)
-    public String health() throws Exception {
-        if (getOrchestrator() == null) {
-            throw new IllegalStateException("Orchestrator service is not ready");
-        }
+    public String health() {
```
Comment (jbonofre) [inline services/api/src/main/java/org/eclipse/osc/services/api/OrchestratorApi.java:39]: Why removing the check on the orchestrator service presence here ?

File: runtime/pom.xml [hunk 2]
```diff
@@ -123,7 +130,7 @@
                             </tags>
                         </to>
                         <container>
-                            <mainClass>org.apache.karaf.minho.boot.Main</mainClass>
+                            <mainClass>org.eclipse.osc.runtime.OscMain</mainClass>
```
Comment (jbonofre) [inline runtime/pom.xml:133]: It means the OscMain will be used only in Docker container. Maybe worth to add in the runtime MANIFEST (Main-Class) in maven-jar-plugin to be able to do `java -jar runtime-1.0-SNAPSHOT.jar`.

**Review by jbonofre — CHANGES_REQUESTED** [review]
Generally speaking, I think it's better:
1. To have dedicated PR per change (avoid to mix multiple changed in one PR)
2. Can you please squash your commits in one (easier to merge) ?

Thanks !

File: services/api/src/main/java/org/eclipse/osc/services/api/OrchestratorApi.java (hunk 1 above)
Comment (swaroopar) [reply services/api/src/main/java/org/eclipse/osc/services/api/OrchestratorApi.java:39]: This check is already there in the getOrchestrator() method. That's why removed it.

Comment (jbonofre) [reply services/api/src/main/java/org/eclipse/osc/services/api/OrchestratorApi.java:39]: Ah yes, good point. I agree then.

**Comment by swaroopar** [comment]
Will create new separate MRs.

**Comment by jbonofre** [comment]
It makes sense. Thanks !

**Pull request closed without merging**